frontend/sw360-portlet/src/main/webapp/html/utils/includes/editAttachments.jspf

```diff
@@ -15,16 +15,19 @@
     Required Imports:       - org.eclipse.sw360.portal.common.PortalConstants
                             - org.eclipse.sw360.datahandler.thrift.attachments.Attachment
                             - org.eclipse.sw360.datahandler.thrift.components.Release
+                            - org.eclipse.sw360.datahandler.thrift.attachments.CheckStatus
 
     Required Beans:         - documentID
                             - documentType
 
+    Required variables:     - hasWritePermissions
+
     Required Stylesheets:   - jquery-ui
                             - jquery-confirm
 
     Included JSPFs:         - utils/includes/attachmentsUpload
 --%>
-
+<%@ taglib prefix="core_t" uri="http://java.sun.com/jsp/jstl/core" %>
```

Comment:
`core_rt` is used as the prefix for JSTL core taglib throughout the codebase. Please adjust for consistency.